packages/typescript-fetch-client/templates/api.hbs

```diff
@@ -125,6 +125,7 @@ export const {{className name}}ApiFetchParamCreator = function (configuration?:
 
 	{{/if}}
 	{{#with requestBody}}
+		{{#ifeq (isContentMultipart defaultContent) false}}
```

Comment:
I think we could use `not` instead of `ifeq false`

Reply:
I tried a few iterations. There's no `not` usage anywhere I can see. Is `{{#unless}}` the thing I want?

Reply:
I've used `{{#unless}}`.